### DISCUSSION TITLE
Specificities of official AWS AMIs and Azure images

### DISCUSSION THREAD

**Discussion by obourdon** [flatcar/Flatcar#697] [Q&A] [answered]
Hello everyone, 

I have been trying to see how the official Flatcar releases (stable channel is my main focus) are produced for AWS and Azure
in terms of potential platform optimisation and customisations but my search so far did not bring anything worth into light

Are there some documents/scripts/CI pipelines publicly available that I could refer to in order to gather this information ?

Many thanks in advance and best regards

**Comment by pothos** [comment, marked as answer]
The customization is done through ebuild files that are installed to the OEM partition.
AWS: https://github.com/flatcar-linux/coreos-overlay/tree/main/coreos-base/oem-ec2-compat
Azure: https://github.com/flatcar-linux/coreos-overlay/tree/main/coreos-base/oem-azure
They contain of a grub.cfg, a base Ignition config, and the package dependencies to put additional software on the OEM partition.
